webmails/rainloop/Dockerfile

```diff
@@ -1,20 +1,21 @@
 FROM php:7.2-apache
 
-RUN apt-get update && apt-get install -y \
-      unzip python3 python3-jinja2
-
 ENV RAINLOOP_URL https://github.com/RainLoop/rainloop-webmail/releases/download/v1.12.1/rainloop-community-1.12.1.zip
 
-RUN rm -rf /var/www/html/ \
+RUN apt-get update && apt-get install -y \
+      unzip python3 python3-jinja2 \
+ && rm -rf /var/www/html/ \
  && mkdir /var/www/html \
  && cd /var/www/html \
  && curl -L -O ${RAINLOOP_URL} \
- && unzip *.zip \
+ && unzip -q *.zip \
  && rm -f *.zip \
  && rm -rf data/ \
  && find . -type d -exec chmod 755 {} \; \
  && find . -type f -exec chmod 644 {} \; \
- && chown -R www-data: *
+ && chown -R www-data: * \
+ && apt-get purge -y unzip \
+ && rm -rf /var/lib/apt/lists
```

Comment:
If you want to make it as lean as possible:
`    && rm -rf /tmp/* /usr/share/doc/* /usr/share/man/* /var/lib/apt/lists/*`

Reply:
In theory you are right but in practice man-pages don't get installed in debian-slim and the docs are only ~200KB

Reply:
Like I said, no major difference, but if you're optimized for size why leave 200KB around?   😉 

Also in my experience `/tmp` can be a nasty one at times so I habitually clean all these folders up at the end of a `RUN`.

Reply:
This could be further improved but currently none of these weighs more than a few kb.